lib/perl5db.pl

```diff
@@ -1538,11 +1538,11 @@ sub _autoflush {
         undef $console;
     }
 
-=item * Windows or MSDOS - use C<con>.
+=item * Windows - use C<con>.
 
 =cut
 
-    elsif ( -e "con" or $^O eq 'MSWin32' ) {
+    elsif ( $^O eq 'MSWin32' and -e "con" ) {
```

Comment:
Does `con` ever _not_ exist on Windows?

Reply:
I have no idea and no way to find out 🤷🏽

Reply:
Yes, it can be removed using DefineDosDevice() function.

Reply:
I don't use windows at all but I think this check:
`elsif ( $^O eq 'MSWin32' and -e "con" ) {`
will always fail unless `-e "con"` does some magic and searches for the DOS special device called CON (short for console). In DOS, CON is a reserved word and can not exist as a file in a straight-forward way. I think the solution will be to simply remove `and -e "con"`. The Unix check further below: `-e "/dev/tty"` makes more sense as Unix is a sensible OS.

Reply:
-e shows CON exists under windows:
```
C:\Users\peter.jones\Downloads\TempData>perl -le "print -e 'CON'"
1
```

Reply:
> I think this check: `elsif ( $^O eq 'MSWin32' and -e "con" ) {` will always fail unless `-e "con"` does some magic and searches for the DOS special device called CON (short for console).

`-e` doesn't do any magic, but the system does. `CON`, `NUL`, `LPT1`, etc are deemed to exist in every directory.

Reply:
got it, thanks